code/game/objects/items/storage/pouch.dm

```diff
@@ -35,6 +38,23 @@
 	..()
 	return user.equip_to_appropriate_slot(src)
 
+/obj/item/storage/pouch/update_icon_state()
+	if(!sprite_slots)
+		return
```

Comment:
this should be setting it to initial probs

Reply:
I thought this would avoid needing to call initial() every time it is opened, but I'll test stuff with initial().